seqr/utils/vcf_utils_tests.py

```diff
@@ -0,0 +1,76 @@
+import mock
```

Comment:
I really dislike having util-file specific tests. It is incredible fragile because we don't really care how the utility function operates, what we actually care about is how the api calls that use those functions operate. If we decide to split a helper function in two or change its function signature in a way that has no impact on any of the functions calling it, we don't want to update a test. Conversely, if the way a utility function is called changes in a breaking way but we are testing the utility function separately from the actual function calls, our tests could easily miss that.

These tests all belong in the `test_validate_anvil_vcf` function in the anvil_workspace_api_tests file. That test and that tets alone should be sufficient to get you to 100% coverage of this new file

While we do have some legacy files that are explicitly testing util functions, we should not add more

Reply:
Okay, I'll merge the util tests to the `test_validate_anvil_vcf`.